src/cli/index.ts

```diff
@@ -1,52 +1,70 @@
 #!/usr/bin/env node
 
-import * as path from 'path';
+import path from 'path';
 
-import * as fs from 'fs-extra';
-import * as globby from 'globby';
+import fs from 'fs-extra';
+import globby from 'globby';
+import chalk from 'chalk';
 
 import format from '../lib';
-import Timer from '../utils/timer';
+import { timer } from '../utils/timer';
+import { pluralize } from '../utils/strings';
 
 import parser from './parse';
 import * as configSearch from './config';
-import logErrorAndExit from './error';
-
-export const handleFile = ({ write, verbose, check }, config) => async (
-  filePath
-) => {
-  const timer = new Timer();
-  timer.start();
-
-  const prevPkg = fs.readJSONSync(filePath, { encoding: 'utf8' });
-
-  const nextPkg = await format(prevPkg, config, filePath);
-
-  if (write) {
-    fs.writeFileSync(filePath, nextPkg, 'utf8');
-  }
+import logError from './error';
+
+export const handleFile =
+  (
+    {
+      write,
+      verbose,
+      check,
+    }: {
+      write: boolean;
+      verbose: boolean;
+      check: boolean;
+    },
+    config
+  ) =>
+  async (filePath: string): Promise<boolean> => {
+    const endTimer = timer()();
+
+    const prevPkg = fs.readJSONSync(filePath, { encoding: 'utf8' });
+    const nextPkg = await format(prevPkg, config, filePath);
+
+    if (write) {
+      fs.writeFileSync(filePath, nextPkg, 'utf8');
```

Comment:
There's a [`writeJson` method exported by `fs-extra`](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/writeJson.md) (also exists in [sync](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/writeJson-sync.md))

Reply:
In this case, the value is a formatted JSON string so we want to write it exactly as it is.